src/core/auth.ts

```diff
@@ -99,6 +99,8 @@ export async function authenticate(jws: GeneralJws, didResolver: DidResolver): P
   // TODO: should we add an explicit check to ensure that there's only 1 signer?, Issue #65 https://github.com/TBD54566975/dwn-sdk-js/issues/65
   const verifier = new GeneralJwsVerifier(jws);
   const { signers } = await verifier.verify(didResolver);
+  if (signers.length!=1)
```

Comment:
style: should wrap all `if` statements with `{... }`, looks like a linting rule needs to be added, feel free to add the rule, but that's optional, I can add it in later.